zinc/src/main/scala/sbt/internal/inc/MixedAnalyzingCompiler.scala

```diff
@@ -299,7 +300,7 @@ object MixedAnalyzingCompiler {
       output: Output,
       outputJarContent: JarUtils.OutputJarContent,
       earlyOutput: Option[Output],
-      earlyAnalysisStore: Option[AnalysisStore],
+      earlyAnalysisStore: Option[XAnalysisStore],
```

Comment:
Why do you want to use an alias for `AnalysisStore`?

If it is not necessary (no name conflict) I would prefer not to use an alias, to avoid having different names for the same thing in different files.

Reply:
I might have confused AnslysisStore and FileAnalysisStore. Let me try to remove the alias.

Reply:
Addressed in https://github.com/sbt/zinc/pull/1350/commits/69c1ab08b7f1dbb8f2b25480b95691881dac6d25